ush/forecast_predet.sh

```diff
@@ -265,4 +265,6 @@
 
   if [[ ! -d "${COM_CHEM_HISTORY}" ]]; then mkdir -p "${COM_CHEM_HISTORY}"; fi
 
+  GOCART_OUTPUT_FH=$(seq -s ' ' "${FHMIN}" "6" "${FHMAX}")
```

Comment:
@aerorahul I thought I tested this but it seems it doesn't work...

Reply:
oh, it's the ensemble fcst, somehow it's treated differently here?

Reply:
I think once #2318 is addressed the line here as-is will be fine, but as it stands now, if you run in ATMA mode with an ensemble forecast FHMIN is 3 instead of 0, and then there isn't a gocart file at hour 3.